mysql.sql

```diff
@@ -0,0 +1,171 @@
+-- MySQL Workbench Forward Engineering
```

Comment:
### It would be better for your experience to write the script by yourself, without script auto-generation